metrics/meteor/meteor.py

```diff
@@ -113,16 +121,40 @@ def _download_and_prepare(self, dl_manager):
 
     def _compute(self, predictions, references, alpha=0.9, beta=3, gamma=0.5):
         if NLTK_VERSION >= version.Version("3.6.5"):
```

Comment:
maybe we can add a comment that before `3.6.5` nltk meteor expected tokenized imputs.